# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1287469545 14400
#      Tue Oct 19 02:25:45 2010 -0400
# Node ID d517388b1c41fe15290ac18397ac28e00c250fa2
# Parent  3cbc0d77db48aec32bcb202d09a036d2cb9cc3b9
Added tag ss-3-3-53 for changeset 6c69a7c39039

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -41,8 +41,9 @@ a5d1e27ee1f4c3fd66c489e83fdb40f92bf1bf43
 739141cde75a22b9ac9b5b327ea30c61fd357748 ss-3-1-52
 bd1b1fe9c6e952202bf5878cd042a168f6df6ddd ss-3-1-53
 3c3cbe8f18e03c73cc34494c8037d8a45a30bf7d ss-3-1-54
 5e276a0b999747b72af34e72eb63f64640da6399 ss-3-1-55
 a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
 74be4b7273e43952e1afcd9ad979d35667c7f82e ss-3-3-50
 79a56d0a6a0d693ebf461c456951b20dfbcde140 ss-3-3-51
 229675bb7647f7e7e499bcb5acacc8eabe7f3697 ss-3-3-52
+6c69a7c39039bccd5a2fe481fcc7f896c82b160b ss-3-3-53
